meta/main.yml

```diff
@@ -5,7 +5,7 @@ galaxy_info:
   company: pro!vision
   issue_tracker_url: https://wcm-io.atlassian.net
   license: Apache
-  min_ansible_version: 2.2
+  min_ansible_version: 2.4
```

Comment:
This needs to be 2.5 as well because this should be always match the ansible version used in the travis build.

Reply:
of course, will be fixed!